.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # This should match the team set up in https://github.com/orgs/opensearch-project/teams and include any additional contributors
-*   @opensearch-project/clients @paulborgermans @shyim
\ No newline at end of file
+*   @VachaShah @dblock @harshavamsi @shyim
```

Comment:
![image](https://user-images.githubusercontent.com/6224096/196652050-96ad9c32-97a6-437a-8cfc-952899c26e22.png)
🤔

Reply:
I added @harshavamsi, but @paulborgermans still has access - if we're removing him let's move him to Emeritus and I can remove acccess?